### PR TITLE
Feature/content updates

### PR DESCRIPTION
**Pull request by jparcher**
**Before:**

Screenshot 2019-04-16 at 15 37 19

**After:**

Screenshot 2019-04-16 at 15 49 26

### CODE REVIEW

File: src/css/site/_form.scss [hunk 1]
```diff
@@ -16,8 +16,8 @@
     height: $input-height;
     line-height: $input-height;
     padding: 0 $space--sm;
-    border: 1px solid #ABB0B2;
-    border-radius: $radius;
+    border: 1px solid $grey;
+    border-radius: 0;
```
Comment (chicgeek) [inline src/css/site/_form.scss]: Can we remove this line entirely now? 

And in the case that the variable isn't used elsewhere, remove it.

File: src/index.html [hunk 2]
```diff
@@ -17,23 +17,31 @@
...
     <link href="css/styles.css" rel="stylesheet">
 </head>
 <body class="body">
     <div class="page">
         <header class="header">
             <h1 class="logo">
-                <img src="img/fes.png" alt="">
-                <span class="logo__title logo__title--primary"><span class="upper">Front End Sheffield</span> <br/>presents:</span>
-                <span class="logo__title  logo__title--secondary">Front End North 2020</span>
+                <img src="img/fen-logo.png" alt="">
```
Comment (chicgeek) [inline src/index.html]: This can be `aria-hidden` for this as it's alongside the text anyway

File: src/index.html [hunk 3]
```diff
@@ -55,16 +63,54 @@ <h2 class="form__header">Want to know more? Get the latest straight to your inbo
...
+                  <use href="#instagram"></use>
+                </svg>
+              </a>
+            </div>
         </main>
     </div>
 
     <script async src="https://www.googletagmanager.com/gtag/js?id=UA-132906563-1"></script>
-        <script>
+      <script>
```
Comment (chicgeek) [inline src/index.html]: Minor: confused as to what's going on with indentation here.

File: src/index.html [hunk 4]
```diff
@@ -55,16 +63,54 @@ <h2 class="form__header">Want to know more? Get the latest straight to your inbo
...
         window.dataLayer = window.dataLayer || [];
         function gtag(){dataLayer.push(arguments);}
         gtag('js', new Date());
 
         gtag('config', 'UA-132906563-1');
     </script>
+
+    <svg width="0" height="0" class="hidden">
+      <symbol viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" id="twitter">
```
Comment (chicgeek) [inline src/index.html]: Were these manually added? If so, okay for now but maybe we can look to inject them later if parcel can manage that.

Comment (jparcher) [reply src/index.html]: Yup, manually added. I will raise an issue ticket to get them built via parcel.

File: src/index.html (hunk 3 above)
Comment (jparcher) [reply src/index.html]: Fixed

File: src/index.html (hunk 2 above)
Comment (jparcher) [reply src/index.html]: Added

File: src/css/site/_form.scss (hunk 1 above)
Comment (jparcher) [reply src/css/site/_form.scss]: Removed

**Pull request merged**